cmd/snap-seccomp/main_test.go

```diff
@@ -238,6 +238,12 @@ faccessat
 		// see libseccomp: _s390x_sock_demux(), _x86_sock_demux()
 		// the -101 is translated to 359 (socket)
 		syscallNr = 359
+	case syscallNr == -10165:
+		// "mknod" on arm64 is not available at all on arm64
+		// only "mknodat" but libseccomp will not generate a
+		// "mknodat" whitelist, it geneates a whitelist with
+		// syscall -10165 (!?!) so we cannot test this.
+		c.Skip("skipping mknod tests on arm64")
 	case syscallNr < 0:
```

Comment:
On a follow up it'd probably make sense to move this into a _default:_ clause, and make the switch on syscallNr itself.